setup.py

```diff
@@ -1,14 +1,31 @@
 import os
-from setuptools import setup, find_packages
+import re
+
+from setuptools import find_packages, setup
 
 here = os.path.abspath(os.path.dirname(__file__))
 
+def get_version(*file_paths):
+    """
+    Extract the version string from the file at the given relative path fragments.
+    """
+    filename = os.path.join(os.path.dirname(__file__), *file_paths)
+    with open(filename, encoding='utf-8') as opened_file:
+        version_file = opened_file.read()
+        version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]",
+                                  version_file, re.M)
+    if version_match:
+        return version_match.group(1)
+    raise RuntimeError('Unable to find version string.')
+
+
+VERSION = get_version("multi_email_field", "__init__.py")
+
+
 setup(
-    name='django-multi-email-field',
-    version='0.6.2',
-    author='Florent Lebreton',
-    author_email='florent.lebreton@makina-corpus.com',
-    url='https://github.com/fle/django-multi-email-field',
+    name='dj-multi-email-field',
```

Comment:
have you checked that this name is available on PyPI?

Reply:
yes it is available.